darwin/utils/utils.py

```diff
@@ -993,13 +994,17 @@ def split_video_annotation(annotation: dt.AnnotationFile) -> List[dt.AnnotationF
         no ``frame_url`` attribute.
     """
     if not annotation.is_video:
-        raise AttributeError("this is not a video annotation")
-
-    if not annotation.frame_urls:
-        raise AttributeError("This Annotation has no frame urls")
-
+        raise AttributeError("This is not a video annotation")
+
+    if not annotation.frame_count and not annotation.frame_urls:
+        raise AttributeError("This Annotation has no frames")
+    urls = (
+        annotation.frame_urls
+        if annotation.frame_urls
+        else [None] * (annotation.frame_count or 1)
```

Comment:
the `or 1` here is just to satisfy type check, the prior assert should already catch that either frame_count or frame_urls should exist but for some reason mypy is not smart enough to pick that up.

Reply:
Can we add a comment somewhere here on why we're chaining this? Long Videos are pretty new, so this code may be a bit confusing without context.